frontend/src/components/ManagePermissionsTable/utils.ts

```diff
@@ -0,0 +1,241 @@
+import { isAxiosError } from "axios";
+import type {
+    AdminRoleAuthGroup,
+    FamAccessControlPrivilegeGetResponse,
+    FamAppAdminGetResponse,
+} from "fam-admin-mgmt-api/model";
+import { formatUserNameAndId } from "@/utils/UserUtils";
+import { formatAxiosError } from "@/utils/ApiUtils";
+import type { PermissionNotificationType } from "@/types/ManagePermissionsTypes";
+import type { FamApplicationUserRoleAssignmentGetSchema } from "fam-app-acsctl-api/model";
+
+export type ConfirmTextType = {
+    role: string;
+    userName: string;
+    appName: string;
+    customMsg?: string;
+};
+
+export const filterList = [
+    "application.app_environment",
+    "application.application_name",
+    "role.client_number.forest_client_number",
+    "role.forest_client.forest_client_number",
+    "role.parent_role.role_name",
+    "role.role_name",
+    "user.email",
+    "user.first_name",
+    "user.last_name",
+    "user.user_name",
+    "user.user_type.description",
+];
+
+/**
+ * Generates the header title text for the table based on the application name.
+ */
+export const getTableHeaderTitle = (
+    appName: string,
+    authGroup: AdminRoleAuthGroup
+): string => {
+    switch (authGroup) {
+        case "FAM_ADMIN":
+            return `${appName} users`;
+        case "APP_ADMIN":
+            return `${appName} users`;
+        case "DELEGATED_ADMIN":
+            return `${appName} delegated administrators`;
+        default:
+            return `${appName} users`;
+    }
+};
+
+/**
+ * Generates a description for the table header section based on the application name.
+ */
+export const getTableHeaderDescription = (
+    appName: string,
+    authGroup: AdminRoleAuthGroup
+): string => {
+    switch (authGroup) {
+        case "FAM_ADMIN":
+            return `
+            This table shows all the users in ${appName} and their permissions levels
```

Comment:
Sorry, maybe I am confused again, the cases and wording seems strange.
Isn't "FAM_ADMIN" can see the "application admin"? 
Also "APP_ADMIN" can see "users" and the "delegated admin"?
I think I misunderstand these switch again.

Reply:
these are for the subtitles in the header sections, i just copied whatever we have in prod

screenshots from test env:
![image](https://github.com/user-attachments/assets/53546866-7798-403b-bd4f-9a718d0da8b6)

![image](https://github.com/user-attachments/assets/f26cddc6-c045-4473-a9a4-16ececcedd52)

Reply:
I see, thanks. I am really confused by these values and text....